pyhelm/repo.py

```diff
@@ -118,7 +120,7 @@ def from_repo(repo_url, chart, version=None, headers=None):
             except:
                 # NOTE(flaper87): Catch requests errors
                 # and untar errors
-                pass
+                raise
```

Comment:
I'm fine with raising the errors we don't catch - it's much better than swallowing them, but if this is the case, why have this try block at all?

Reply:
I was thinking the same thing actually. Are there any errors that shouldn't be raised? If not I'll remove the try block.

Reply:
I don't think there are any exceptions that should be swallowed. IMHO - delete it.